app/(white_navbar)/explore/RetroPGF4/_component/Tab/component/InputAndFilterBadgeholderTab.tsx

```diff
@@ -0,0 +1,211 @@
+import { Dispatch, FC, SetStateAction } from 'react'
+import {
+  CheckBoxStateBadgeholderType,
+  CheckBoxStateType,
+  ExploreRoundState,
+} from '../../ExploreRoundType'
+import Input from '@/app/component/Input/Input'
+import { badgeholderStatus, category, multiplyOpenSource } from '../../Text'
+import Search from '@carbon/icons-react/lib/Search'
+import Grid from '@carbon/icons-react/lib/Grid'
+import List from '@carbon/icons-react/lib/List'
+import SettingsAdjust from '@carbon/icons-react/lib/SettingsAdjust'
+import SortExploreRound from '../../SortExploreRound'
+import SortBadgeholder from './SortBadgeholder'
+
+interface IInputAndFilterBadgeholderTabProps {
+  search: string
+  setCheckBox: Dispatch<SetStateAction<any>>
+  setSearch: Dispatch<SetStateAction<string>>
+  setState: Dispatch<SetStateAction<any>>
```

Comment:
Can you put a type of state on `setCheckBox` and `setState` instead of any so we don't have to put any everywhere in the file like Dispatch<SetStateAction<StateType>>